relay-server/src/metrics_extraction/transactions/mod.rs

```diff
@@ -736,6 +740,131 @@ fn extract_captured_substring<'a>(string: &'a str, pattern: &'a Lazy<Regex>) ->
     None
 }
 
+/// Returns the category of a span from its operation. The mapping is available in:
+/// <https://develop.sentry.dev/sdk/performance/span-operations/>
+fn span_op_to_category(op: &str) -> Option<&str> {
+    Some({
```

Comment:
It's not really readable, maybe better would be to assign these `if`s to the variable and return it at the end, like 

```rust

  let category = if op.starts_with("mark") {
...

  // at the end of the function 
  Some(category)
}
```